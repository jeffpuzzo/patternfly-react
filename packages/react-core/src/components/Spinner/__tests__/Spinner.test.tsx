import * as React from 'react';
import { render } from '@testing-library/react';
import { Spinner } from '../Spinner';

test('simple spinner', () => {
  const view = render(<Spinner />);
  expect(view.container.outerHTML).toMatchSnapshot();
});

test('small spinner', () => {
  const view = render(<Spinner size="sm" />);
  expect(view.container.outerHTML).toMatchSnapshot();
});

test('medium spinner', () => {
  const view = render(<Spinner size="md" />);
  expect(view.container.outerHTML).toMatchSnapshot();
});

test('large spinner', () => {
  const view = render(<Spinner size="lg" />);
  expect(view.container.outerHTML).toMatchSnapshot();
});

test('extra large spinner', () => {
  const view = render(<Spinner size="xl" />);
  expect(view.container.outerHTML).toMatchSnapshot();
});
