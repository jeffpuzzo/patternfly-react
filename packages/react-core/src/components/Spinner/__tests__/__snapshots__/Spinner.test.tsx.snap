// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extra large spinner 1`] = `"<div><span class=\\"pf-c-spinner pf-m-xl\\" role=\\"progressbar\\" aria-valuetext=\\"Loading...\\" aria-label=\\"Contents\\"><span class=\\"pf-c-spinner__clipper\\"></span><span class=\\"pf-c-spinner__lead-ball\\"></span><span class=\\"pf-c-spinner__tail-ball\\"></span></span></div>"`;

exports[`large spinner 1`] = `"<div><span class=\\"pf-c-spinner pf-m-lg\\" role=\\"progressbar\\" aria-valuetext=\\"Loading...\\" aria-label=\\"Contents\\"><span class=\\"pf-c-spinner__clipper\\"></span><span class=\\"pf-c-spinner__lead-ball\\"></span><span class=\\"pf-c-spinner__tail-ball\\"></span></span></div>"`;

exports[`medium spinner 1`] = `"<div><span class=\\"pf-c-spinner pf-m-md\\" role=\\"progressbar\\" aria-valuetext=\\"Loading...\\" aria-label=\\"Contents\\"><span class=\\"pf-c-spinner__clipper\\"></span><span class=\\"pf-c-spinner__lead-ball\\"></span><span class=\\"pf-c-spinner__tail-ball\\"></span></span></div>"`;

exports[`simple spinner 1`] = `"<div><span class=\\"pf-c-spinner pf-m-xl\\" role=\\"progressbar\\" aria-valuetext=\\"Loading...\\" aria-label=\\"Contents\\"><span class=\\"pf-c-spinner__clipper\\"></span><span class=\\"pf-c-spinner__lead-ball\\"></span><span class=\\"pf-c-spinner__tail-ball\\"></span></span></div>"`;

exports[`small spinner 1`] = `"<div><span class=\\"pf-c-spinner pf-m-sm\\" role=\\"progressbar\\" aria-valuetext=\\"Loading...\\" aria-label=\\"Contents\\"><span class=\\"pf-c-spinner__clipper\\"></span><span class=\\"pf-c-spinner__lead-ball\\"></span><span class=\\"pf-c-spinner__tail-ball\\"></span></span></div>"`;
